prime-router/src/main/kotlin/azure/WorkflowEngine.kt

```diff
@@ -184,7 +184,7 @@ class WorkflowEngine(
         // Send immediately.
         val nextEvent = ReportEvent(Event.EventAction.SEND, reportId, at = null)
         db.transact { txn ->
-            val task = db.fetchAndLockTask(reportId, txn)
+            val task = db.fetchAndLockTask(reportId, txn) // variable is not being used.
```

Comment:
Then remove

Reply:
@carlosfelix2 - this is one example of a variable being unused but being locked by the DB

Reply:
Ok, so is this one fine like this?

Reply:
Yup

Reply:
Follow up: made the changes.